runtime-modules/content/src/lib.rs

```diff
@@ -2348,6 +2353,39 @@ decl_module! {
             }
         }
 
+        /// Claims an accumulated channel reward for a council.
+        #[weight = 10_000_000] // TODO: adjust weight
+        pub fn claim_council_reward(
+            origin,
+            channel_id: T::ChannelId,
+        ) {
+            let channel = Self::ensure_channel_exists(&channel_id)?;
+
+            ensure_actor_authorized_to_claim_council_reward::<T>(origin, &channel.owner)?;
+
+            ensure!(
+                channel.transfer_status == ChannelTransferStatus::NoActiveTransfer,
+                Error::<T>::InvalidChannelTransferStatus
+            );
+
+            //
+            // == MUTATION SAFE ==
+            //
+
+            let channel_account_id = ContentTreasury::<T>::account_for_channel(channel_id);
+            let reward: BalanceOf<T> = Balances::<T>::usable_balance(&channel_account_id);
+
+            if !reward.is_zero(){
+                let _ = Balances::<T>::slash(&channel_account_id, reward);
+
+                let budget = T::CouncilBudgetManager::get_budget();
```

Comment:
Seems it would be useful to just have a  `add_to_budget` in the `CouncilBudgetManager`, to avoid this reading and safe addition and setting in client code.

Reply:
We will have this method as `increase_budget` in the `ephesus` branch.